plugins/inputs/ceph/ceph.go

```diff
@@ -44,7 +46,8 @@ type Ceph struct {
 	GatherAdminSocketStats bool   `toml:"gather_admin_socket_stats"`
 	GatherClusterStats     bool   `toml:"gather_cluster_stats"`
 
-	Log telegraf.Logger `toml:"-"`
+	Log        telegraf.Logger          `toml:"-"`
+	SchemaMaps map[socket]perfSchemaMap `toml:"-"`
```

Comment:
Please do not export this internal member
```suggestion
	schemaMaps map[socket]perfSchemaMap
```